doc/api/documentation.markdown

```diff
@@ -66,3 +66,19 @@ Every HTML file in the markdown has a corresponding JSON file with the
 same data.
```

Comment:
Anyways, I'll continue my crusade on docs today and tomorrow. :)

On Sat, Feb 13, 2016, 08:59 thefourtheye <notifications@github.com> wrote:

> In doc/api/documentation.markdown
> <https://github.com/nodejs/node/pull/5073#discussion_r52824032>:
>
> > @@ -66,3 +66,19 @@ Every HTML file in the markdown has a corresponding JSON file with the
> >  same data.
>
> Ah my bad. I just saw few linkable text on this page and reached out for
> your help. Perhaps we can leave this page as nothing much is needed here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/5073/files#r52824032>.
>
-- 
Sincerely,
Alexander Makarenko